WatchApp Extension/Assets.xcassets/Complication.complicationset/Graphic Extra Large.imageset/Contents.json

```diff
@@ -0,0 +1,28 @@
+{
+  "images" : [
```

Comment:
Similarly here: this, I think, is only different because of locally building on Xcode 12.5.

Reply:
This looks new?